<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>_DictionaryGUIClass</class>
 <widget class="QMainWindow" name="_DictionaryGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>_DictionaryGUI</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="welcome_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>461</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 170, 0);</string>
    </property>
    <property name="text">
     <string>Welcome to Oxford Dictionary</string>
    </property>
   </widget>
   <widget class="QLabel" name="version">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>300</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Version 1.0 - Release: 26/11/2018</string>
    </property>
   </widget>
   <widget class="QLabel" name="author">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 255);</string>
    </property>
    <property name="text">
     <string>Author: Nguyen Khanh Toan - 17CNTN- HCMUS</string>
    </property>
   </widget>
   <widget class="QLabel" name="findIcon">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/newPrefix/img_388871.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;
font-family: &quot;Open sans&quot;;
background: #262626;
border: none;
font-size: 16px;
font-weight: bolder;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/remove.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="meaning">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>441</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="notification">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>511</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Georgia</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Notification: Nothing new!</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="meaningAdd">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>441</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAdd">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;
font-family: &quot;Open sans&quot;;
background: #262626;
border: none;
font-size: 16px;
font-weight: bolder;
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditRemove">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;
font-family: &quot;Open sans&quot;;
background: #262626;
border: none;
font-size: 16px;
font-weight: bolder;
</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="meaningRemove">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>441</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="okRemoveButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditChange">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;
font-family: &quot;Open sans&quot;;
background: #262626;
border: none;
font-size: 16px;
font-weight: bolder;
</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="meaningChange">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>441</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="okChangeButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>190</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>607</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuImport">
    <property name="title">
     <string>Import</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuLookUp">
    <property name="title">
     <string>Look Up</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit_Dictionary">
    <property name="title">
     <string>Edit Dictionary</string>
    </property>
    <addaction name="actionAddNewWord"/>
    <addaction name="actionDeleteWord"/>
    <addaction name="actionChangeMeaning"/>
   </widget>
   <addaction name="menuImport"/>
   <addaction name="menuExport"/>
   <addaction name="menuLookUp"/>
   <addaction name="menuEdit_Dictionary"/>
  </widget>
  <action name="actionAddNewWord">
   <property name="text">
    <string>AddNewWord</string>
   </property>
  </action>
  <action name="actionDeleteWord">
   <property name="text">
    <string>DeleteWord</string>
   </property>
  </action>
  <action name="actionChangeMeaning">
   <property name="text">
    <string>ChangeMeaning</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="_DictionaryGUI.qrc"/>
 </resources>
 <connections/>
</ui>
